source/_posts/setup-tailwindcss_parcel_npm.md

```diff
@@ -0,0 +1,237 @@
+&& npm i -g parcel-bundler
+```
+
+Description by line:
+
+1. Create new directory, change `new_tailwind_project` to whaterver name you want.
+1. Create a `package.json` file
+1. Add required development dependencies with explicit version numbers, for example tailwind will be: `0.6.4`
+1. Create an empty postcss configuration file.
+1. Generate tailwind configuration file.
+1. Create `src` folder and the main CSS file where you can define your custom css.
+1. Make sure that parcel is installed.
+
+[Follow common configuration](#common-configuration) and then come back to generate your files.
+
+** HTML **
+
+Because Parcel works by parsing all your files, you have to add the following line to `src/index.html` after `<title>`:
+
+```html
+<!-- ... -->
+<title>Tailwindcss starter</title>
+<!-- add the following line -->
+<link rel="stylesheet" href="./tailwind.css">
+<!-- ... -->
+```
+
+In your terminal type:
+
+```
+>open http://localhost:1234 && parcel src/index.html
+```
+
+Wait for the build in your terminal and the browser should auto-refresh when done. Now you can start modifying the HTML and continue your work.
+
+### npm
+
+You can copy paste the following lines and paste them into your terminal, and explanation will come afterwards:
+
+```
+mkdir new_tailwind_project && cd $_ \
+&& npm init -y \
+&& npm install -E -D postcss-cli tailwindcss autoprefixer cssnano @fullhuman/postcss-purgecss \
+&& touch postcss.config.js \
+&& ./node_modules/.bin/tailwind init \
+&& mkdir src && touch src/tailwind.css && touch src/index.html
+```
+
+Description by line:
+
+1. Create new directory, change `new_tailwind_project` to whaterver name you want.
+1. Create a `package.json` file
+1. Add required development dependencies with explicit version numbers, for example tailwind will be: `0.6.4`
+1. Create an empty postcss configuration file.
+1. Generate tailwind configuration file.
+1. Create `src` folder and the main CSS file where you can define your custom css.
+
+Then modify you `package.json` file to include a npm script to be able to build your css, you will set your source from `tailwind.css` and output to `src/style.css`:
+
+```json
+...
+scripts: {
+  "css": "postcss src/tailwind.css -o src/style.css"
+}
+...
+```
+
+[Follow common configuration](#common-configuration) and then come back to generate your files.
+
+** HTML ** 
+Before generating all your files, add the following line after `<title>` in `src/index.html`:
+
+```html
+<!-- ... -->
+<title>Tailwindcss starter</title>
+<!-- add the following line -->
+<link rel="stylesheet" href="style.css"> 
+<!-- ... -->
+```
+
+As a last step, build your css and open `index.html`:
+
+```
+> npm run css && open src/index.html
+```
+
+Hopefully now you can start customizing your live-reload or `dist` folder.
+
+### <a name="common-configuration"></a> Common configuration 
+
+** Postcss **
+
+Modify your `postcss.config.js` like this:
+
+```js
```

Comment:
little quesiton, using `js` works the same as `javascript` for highlighting code?